client/store/rootReducer.ts

```diff
@@ -1,9 +1,11 @@
 import AssessmentSlice from './assessment/slice'
 import LoginSlice from './login/slice'
 import UserSlice from './user/slice'
+import UiSlice from './ui/slice'
 
 export default {
   assessment: AssessmentSlice,
   login: LoginSlice,
   user: UserSlice,
+  ui: UiSlice,
```

Comment:
store should be ui/navigation.

export actions/hooks/reducer etc from store/ui/navigation 
e.g. ```import {NavigationActions} from '@store/ui/navigation'```